testing.rst

```diff
@@ -194,6 +194,25 @@ code to production:
             $twig->strictVariables(true);
         };
 
+Another example for HTTP sessions:
```

Comment:
I don't think this addition adds much, let's remove it?

If you think showcasing `MockFileSessionStorage` in the docs (instead of just the recipe) is worth it, I would suggest adding a small section like "Using sessions in tests" in the session docs instead.

Reply:
It was more for people not using the framework, but it looks weird.
I'm going to remove it.